11-Backbonejs/04-Backbone-Models.md

```diff
@@ -291,6 +291,7 @@ The reason has to do with how DOM event listeners work. Remember that our `compl
 ```javascript
 this.listElement.empty();
 ```
+You will also need to update the TaskListView `getInput` function to use a Task model.
```

Comment:
I don't believe this is true. You should be able to `add` a raw JavaScript object to a collection, and Backbone will figure it out.